### PR TITLE
Add gpu and inf1 test

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/download_models.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ aarch_models_urls=(
 )
 
 inf_models_urls=(
-"https://resources.djl.ai/test-models/pytorch/resnet18_inf1.tar.gz"
+"https://djl-ai.s3.amazonaws.com/resources/test-models/pytorch/resnet18_inf1_1_11.tar.gz"
```
Comment (lanking520) [inline tests/integration/download_models.sh:22]: resources.djl.ai == djl-ai.s3.amazonaws.com/resources

**Review by lanking520 — APPROVED** [review]

File: tests/integration/download_models.sh (hunk 1 above)
Comment (sindhuvahinis) [reply tests/integration/download_models.sh:22]: Okay. (y)

**Review by lanking520 — APPROVED** [review]
